pkgs/development/python-modules/qiskit-aqua/default.nix

```
@@ -0,0 +1,104 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, cvxopt
+, dlx
+, docplex
+, fastdtw
+, h5py
+, networkx
+, numpy
+, psutil
+, qiskit-ignis
+, qiskit-terra
+, quandl
+, scikitlearn
+  # Check Inputs
+, parameterized
+, pytestCheckHook
+, qiskit-aer
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-aqua";
+  version = "0.6.5";
+
+  disabled = pythonOlder "3.5";
+
+  # Pypi's tarball doesn't contain tests
+  src = fetchFromGitHub {
+    owner = "Qiskit";
+    repo = pname;
```

```suggestion
    repo = "qiskit-aqua";
```

This is an "accidental similarity" (my own term, there's probably a real name for this but I haven't found it yet). While `repo` and `pname` are the same here, they don't necessarily change together. So it would be better to just repeat the name here.

---

ack

---

@timokau re "accidental similarity": I suggest "coincidental similarity"? Or "naming hash collision" :D